output/json/json.go

```diff
@@ -132,11 +130,13 @@ func (o *Output) Stop() error {
 
 // SetThresholds receives the thresholds before the output is Start()-ed.
 func (o *Output) SetThresholds(thresholds map[string]metrics.Thresholds) {
-	ths := make(map[string][]*metrics.Threshold)
+	if thresholds == nil || len(thresholds) < 1 {
```

Comment:
```suggestion
	if len(thresholds) == 0 {
```

is enough, but I wonder if we even need it :thinking: does `make` panic on length being 0 ?